src/fmod_server.cpp

```diff
@@ -879,17 +885,28 @@ float FmodServer::get_global_parameter_by_name(const String& parameterName) {
     return value;
 }
 
-void FmodServer::set_global_parameter_by_id(const Array& idPair, const float value) {
-    if (idPair.size() != 2) {
+void FmodServer::set_global_parameter_by_id(const Array& id_pair, const float value) {
+    if (id_pair.size() != 2) {
         GODOT_LOG_ERROR("FMOD Sound System: Invalid parameter ID")
         return;
     }
     FMOD_STUDIO_PARAMETER_ID id;
-    id.data1 = idPair[0];
-    id.data2 = idPair[1];
+    id.data1 = id_pair[0];
```

Comment:
Would be the opportunity to get rid of this awful array of 2 integers and simply use a long, just like we already do with regular event parameters.
They both use FMOD_STUDIO_PARAMETER_ID after all, so currently the use of ids is not consistent accros the plugin.